restonomer-core/src/it/resources/restonomer_context/checkpoints/transformation/checkpoint_add_prefix_to_column_names_transformation.conf

```diff
@@ -0,0 +1,25 @@
+name = "checkpoint_add_prefix_to_column_names_transformation"
+
+data = {
+  data-request = {
+    url = "http://localhost:8080/add-prefix-to-column-names-transformation"
+  }
+
+  data-response = {
+    body-format = "JSON"
+
+    transformations = [
+      {
+        type = "add-prefix-to-column-names"
+        prefix = "test_"
```

Comment:
Dont expect user to provide underscore.
Add undersore after prefix in the code itself.

Reply:
Underscore will now be getting added automatically, separating prefix and actual column name. User don't need to provide it.